Portfolio2.css

```diff
@@ -0,0 +1,222 @@
+* {
+	margin:0;
+	padding: 0;
+	
+}
+
+body {
+	background-image: url("https://wallpaperaccess.com/full/1128684.jpg");
+	background-size: cover;
+	
+	
+ }
+
+header {
+	display: inline-flex; 
+	position: fixed; 
+	margin-left: 1200px;
+	position:fixed;
+	margin-top:-180px;
+}
+
+.menue ul li {
+	display: inline-flex;
+	list-style: none;
+	cursor:pointer;
+	padding:25px;
+	
+}
+
+.logo img {
+	width: 90px;
+	margin-top: -7px;
+	margin-right: 48px;
+	
+}
+
+.active {
+	color: grey
+}
+.activ {
+	color: grey;
+}
+
+.acti {
+	color: grey;
+}
+
+#headertwo {
+        background-color: none;
+    	width: 100%;
```

Comment:
Take this out, because the div is block by default it should automatically take up all available width, this will also fix the horizontal scroll bar